angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -74,18 +870,22 @@ export const standout = () => ({
         sprk-o-Stack__item
         sprk-c-Card__content
         sprk-o-Stack
-        sprk-o-Stack--medium">Standout Card Content</div>
+        sprk-o-Stack--medium"
+      >
+        <p class="sprk-TypeBodyTwo">Standout Card</p>
```

Comment:
missing `sprkText`

Reply:
@bobwise that is a legacy story, the old code API.